crates/orchestrator/src/cron/mod.rs

```diff
@@ -0,0 +1,50 @@
+use std::time::Duration;
+
+use async_trait::async_trait;
+use lazy_static::lazy_static;
+
+use crate::queue::job_queue::{WorkerTriggerMessage, WorkerTriggerType};
+use crate::setup::SetupConfig;
+
+pub mod event_bridge;
+
+lazy_static! {
+    pub static ref CRON_DURATION: Duration = Duration::from_mins(1);
+    pub static ref TARGET_QUEUE_NAME: String = String::from("madara_orchestrator_worker_trigger_queue");
```

Comment:
can we use the same constant here and the queue file
also add TODO as you mentioned here that we can take this from clap later on

Reply:
we don't have identical constant with `TARGET_QUEUE_NAME` but I have added the comment here.